ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-openshift-v4.yaml

```diff
@@ -0,0 +1,36 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: plugin-base
+    namespace: azure
+    tag: latest
+binary_build_commands: make all
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: test-base
+    namespace: azure
+    tag: latest
+canonical_go_repository: github.com/openshift/openshift-azure
+images:
+- dockerfile_path: images/azure/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/openshift-azure/azure
+  to: azure
+promotion:
+  cluster: https://api.ci.openshift.org
+  name: azure-plugins
+  namespace: azure
+resources:
+  '*':
+    requests:
+      cpu: 10m
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v3.11
```

Comment:
is this correct? maybe v4?

Reply:
yes, I tried to understand why it fails. Apparently because of https://github.com/openshift/release/pull/4044